Code-Hills/codehills-dev-profile-fe
# Ft user role

# Change User role

## Description
This pull request adds the following user functionalities to the project:
- Admin can view a list of users
- Admin can change a user role

## Screenshots
<img width="1680" alt="Screenshot 2023-05-16 at 2 20 02 PM" src="https://github.com/Code-Hills/codehills-dev-profile-fe/assets/55650162/e1ee07a1-174e-41ba-9bd2-efdc92cb2bec">
<img width="1680" alt="Screenshot 2023-05-16 at 2 20 13 PM" src="https://github.com/Code-Hills/codehills-dev-profile-fe/assets/55650162/59aa8c07-98bb-4264-95e8-416fdb970cff">




## Testing
The following tests were added to ensure the new admin functionality works as expected:
- checkout to `ft-user-role` branch
- start app by `yarn dev`
- visit page in browser at: localhost:[port]/user

## Known Issues
There are currently no known issues with this pull request.
